vendor/gopkg.in/mgo.v2/session.go

```diff
@@ -2535,6 +2535,13 @@ func (c *Collection) Insert(docs ...interface{}) error {
 	return err
 }
 
+// InsertUnordered is the same as Inesrt but has the "unordered" or
```

Comment:
How are we going to add this back if we ever revendor mgo?

Reply:
I don't expect that we'll ever revendor, and if we do it'll break compile so we'll be able to handle this. 

(Its also true that because we're using the 10gen/mgo fork that revendering mgo is really hard.) 

This is fine as is.